timrlink.net.CLI/Actions/TaskImportAction.cs

```diff
@@ -31,8 +31,8 @@ public override async Task Execute()
         {
             var tasks = TaskService.FlattenTasks(await TaskService.GetTaskHierarchy());
             var taskTokenDictionary = tasks.ToTokenDictionary();
-            var existingTasks = taskTokenDictionary.ToDictionary(pair => pair.Key, pair => pair.Value);
```

Comment:
I now know again why we added this here, because the `taskTokenDictionary` gets modified by `AddTaskTreeRecursive()` and the `existingTasks` should be kept unmodified.

Otherwise the first if statement in `AddOrUpdateTask` doesn't work.

You could try this by adding an existing Task to the very first test case. That one must not get updated like in the second test case.